### ISSUE TITLE
Define production units for all lands/waters products

### ISSUE THREAD

**Issue by shawnbot** [18F/doi-extractives-data#1396]
For some EIA commodities, such as renewables, the units are unknown. These are the products:

- [ ] `All Other Renewables`
- [ ] `Biomass (total)`
- [ ] `Conventional Hydroelectric`
- [ ] `Geothermal`
- [ ] `Other biomass`
- [ ] `Solar`
- [ ] `Wind`
- [ ] `Wood and wood-derived fuels`

We should probably create a mapping of these , e.g. in `_data/production_units.yml`:

```yaml
'Biomass (total)': 'tons'
'Wind': 'kilowatt hours'
```

Then, in our templates we would get the units with:

```html
{% assign units = product.units | default: site.data.production_units[product.name] %}
```

**Comment by shawnbot** [comment]
I've added a task list to this issue description with the product names for which we don't have units in both the all lands and federal production data.

**Comment by shawnbot** [comment]
Another thing to consider: Should we have shorthand notation for some units, like `kWh` for kilowatt-hours? If so, our units definition data might look like this instead:

```yaml
products:
  'Wind': 'kilowatt-hours'

shorthand:
  'kilowatt-hours': 'kWh'

**Comment by meiqimichelle** [comment]
@shawnbot is this something that we should resolve before your time gets split?

**Comment by shawnbot** [comment]
Nope, I think y'all can handle this if I'm not able to.

**Comment by coreycaitlin** [comment]
Some of these products would also make more sense with the word "energy" after them (i.e. "hydroelectric energy").

**Comment by shawnbot** [comment]
So I think @coreycaitlin hit upon a pretty good justification for having something like this instead in, say, `_data/production_units.yml` (or whatever):

```yaml
products:
  Hydroelectric:
    name: Hydroelectric Energy
    units: kilowatt-hours

abbreviations:
  kilowatt-hours: kWh
```

Then, in places where we use `product_name` in our Jekyll templates now, we would do something like this:

```html
{% assign product_name = product[1].name %}
{% assign units = product[1].units %}
{% assign units_abbr = units %}
{% if product_name in site.data.production_units %}
  {% assign _product = site.data.production_units[product_name] %}
  {% assign product_name = _product.name | default: product_name %}
  {% assign units = _product.units | default: units | default: 'units' %}
  {% assign units_abbr = site.data.production_units.abbreviations[units] | default: units_abbr %}
{% endif %}
```

**Comment by meiqimichelle** [comment]
Perhaps as a first step, we need to understand where on the page we want the units to display to make sure it can be the same words throughout, or if we need to adjust for each? For example, right now there aren't units in the bar chart production number (ie, next to the number). We probably want them there?? @coreycaitlin maybe you already have this somewhere -- I know you've been thinking about units.

**Comment by gemfarmer** [comment]
@ericronne do we know where we want the units on the bar charts and maps?

@coreycaitlin do you have an idea for how we should handle production units?

**Comment by ericronne** [comment]
Current plan was to list the units after the number, see #1556 
We need to confirm that this will work with all charts, tho

**Comment by ericronne** [comment]
Maps...
![image](https://cloud.githubusercontent.com/assets/9259185/16595448/c09c6004-42b4-11e6-8c61-80cefab7a678.png)

**Comment by coreycaitlin** [comment]
@ericronne Are you thinking we'd show units after just the first one? Or all of them? I'd be fine with putting it in the map/legend title, too.

@gemfarmer and I talked about this a bit today and came up with these examples:

![screen shot 2016-07-05 at 2 12 02 pm](https://cloud.githubusercontent.com/assets/1450354/16600924/78afa38c-42be-11e6-9dbb-b569675bcba0.png)

![screen shot 2016-07-05 at 2 13 21 pm](https://cloud.githubusercontent.com/assets/1450354/16600927/7ae67d2e-42be-11e6-8390-450f1c436b68.png)

![screen shot 2016-07-05 at 2 21 22 pm](https://cloud.githubusercontent.com/assets/1450354/16600887/44ac4f04-42be-11e6-98a8-9cf6f324b6ab.png)

**Comment by coreycaitlin** [comment]
I added an issue to cover some redundancies in the all-lands production product listing: #1574 

I've taken a first pass through trying to sort out some unit names and units, and here are my questions!

- One dataset uses `Coal (short tons)` and another uses `Coal (tons)`. Do both actually use short tons, or are they actually using different units?
- Re: `Conventional Hydroelectric`: Can we just call this "Hydroelectric"? Do we have any other kinds of hydroelectric energy?
- One dataset uses `Gas (mcf)` and another uses `Natural Gas (mcf)` Do these both refer to natural gas, or is there a difference? (This also came up in one usability test.)
- Geothermal! @mentastc, I might need some help understanding the intricacies of the 6 different ways we list geothermal.

@meiqimichelle or @mentastc, let me know if any of these question are more complicated than I figured and would be best to talk through in person.
